### PR TITLE
clearing the db

### PR DESCRIPTION
**Pull request by farhanp1502**

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -33,7 +33,7 @@
     "@capacitor/status-bar": "6.0.0",
     "@ionic/angular": "^7.0.0",
     "@zxing/browser": "^0.1.5",
-    "authentication_frontend_library": "^0.0.62",
+    "authentication_frontend_library": "^0.0.63",
```
Comment (Vinod-V3) [inline package.json:36]: Why is it changed to 0.0.63?
Any changes or updates done?

File: src/app/home/home.page.ts [hunk 2]
```diff
@@ -32,6 +32,7 @@ export class HomePage {
   @ViewChild('bannerTemplate') bannerTemplate!: TemplateRef<any>;
   @ViewChild('solutionTemplate') solutionTemplate!: TemplateRef<any>;
   @ViewChild('recommendationTemplate') recommendationTemplate!: TemplateRef<any>;
+  sharePopupHandler:any;
```
Comment (Vinod-V3) [inline src/app/home/home.page.ts:35]: Give relevant names based on the action done
Its not a sharePopupHandler

File: package.json (hunk 1 above)
Comment (farhanp1502) [reply package.json:36]: the emit from the logout is been done in the this version

File: src/app/home/home.page.ts (hunk 2 above)
Comment (farhanp1502) [reply src/app/home/home.page.ts]: done

**Pull request merged**